### PR TITLE
Add mint tests

### PR DESCRIPTION
**Pull request by anataliocs**
This PR adds unit tests written in JS in `test.js` that will test smart contract functionality.

Testing steps:
```
truffle test
```

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -9,35 +9,77 @@ let infura1155NFTInstance;
 contract('Infura721aNFTContract', function (accounts) {
 
     it("Contract deployment", function () {
-        //Fetching the contract instance of our smart contract
-        return Infura721aNFT.deployed().then(function (instance) {
+        return Infura721aNFT.deployed({from: accounts[0]}).then(function (instance) {
```
Comment (emilyJLin95) [inline test/test.js]: is there a reason you don't just do `.deployed()`? it should be deploying from `accounts[0]`

File: test/test.js [hunk 2]
```diff
@@ -9,35 +9,77 @@ let infura1155NFTInstance;
...
             infura721aNFTInstance = instance;
             assert(infura721aNFTInstance !== undefined, 'Infura721aNFT contract should be defined');
         });
     });
 
+    it("Should mint Infura721aNFT token", function () {
+        const toAddress = accounts[2];
```
Comment (emilyJLin95) [inline test/test.js]: jw why are you using `accounts[2]` and not `accounts[1]`

File: test/test.js (hunk 1 above)
Comment (anataliocs) [reply test/test.js]: Fixed!

File: test/test.js (hunk 2 above)
Comment (anataliocs) [reply test/test.js]: B/c I forgot to start counting from 0? 😆

**Review by emilyJLin95 — APPROVED** [review]
Lgtm

**Pull request merged**